torch/utils/data/datapipes/iter/web.py

```diff
@@ -0,0 +1,34 @@
+from io import IOBase
+from typing import Tuple
+from urllib.error import HTTPError, URLError
+import urllib.request as urllib
+from torch.utils.data import IterDataPipe
+
+
+class WebIterDataPipe(IterDataPipe[Tuple[str, IOBase]]):
```

Comment:
Can we change name to HTTPReader? But, relying on @VitalyFedyunin about the naming of DataPipe.

Reply:
Agree about naming

Reply:
fyi, cool, name changed